packages/website/ts/components/dropdowns/developers_drop_down.tsx

```diff
@@ -0,0 +1,188 @@
+import { colors } from '@0xproject/react-shared';
+import * as _ from 'lodash';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { DropDown } from 'ts/components/ui/drop_down';
+import { Deco, Key, WebsitePaths } from 'ts/types';
+import { constants } from 'ts/utils/constants';
+import { Translate } from 'ts/utils/translate';
+
+interface KeyToLinkInfo {
+    [key: string]: LinkInfo;
+}
+
+interface LinkInfo {
+    link: string;
+    shouldOpenNewTab: boolean;
+}
+
+const gettingStartedKeyToLinkInfo1: KeyToLinkInfo = {
+    [Key.BuildARelayer]: {
+        link: `${WebsitePaths.Wiki}#Build-A-Relayer`,
+        shouldOpenNewTab: false,
+    },
+    [Key.IntroTutorial]: {
+        link: `${WebsitePaths.Wiki}#Create,-Validate,-Fill-Order`,
+        shouldOpenNewTab: false,
+    },
+};
+const gettingStartedKeyToLinkInfo2: KeyToLinkInfo = {
+    [Key.TradingTutorial]: {
+        link: `${WebsitePaths.Wiki}#Find,-Submit,-Fill-Order-From-Relayer`,
+        shouldOpenNewTab: false,
+    },
+    [Key.EthereumDevelopment]: {
+        link: `${WebsitePaths.Wiki}#Ethereum-Development`,
+        shouldOpenNewTab: false,
+    },
+};
+const popularDocsToLinkInfos: KeyToLinkInfo = {
+    [Key.ZeroExJs]: {
+        link: WebsitePaths.ZeroExJs,
+        shouldOpenNewTab: false,
+    },
+    [Key.Connect]: {
+        link: WebsitePaths.Connect,
+        shouldOpenNewTab: false,
+    },
+    [Key.SmartContract]: {
+        link: WebsitePaths.SmartContracts,
+        shouldOpenNewTab: false,
+    },
+};
+const usefulLinksToLinkInfo: KeyToLinkInfo = {
+    [Key.Github]: {
+        link: constants.URL_GITHUB_ORG,
+        shouldOpenNewTab: true,
+    },
+    [Key.Whitepaper]: {
+        link: WebsitePaths.Whitepaper,
+        shouldOpenNewTab: true,
+    },
+    [Key.Sandbox]: {
+        link: constants.URL_SANDBOX,
+        shouldOpenNewTab: true,
+    },
+};
+
+interface DevelopersDropDownProps {
+    translate: Translate;
+    menuItemStyles: React.CSSProperties;
+    menuIconStyle: React.CSSProperties;
+}
+
+interface DevelopersDropDownState {}
+
+export class DevelopersDropDown extends React.Component<DevelopersDropDownProps, DevelopersDropDownState> {
+    public render(): React.ReactNode {
+        const activeNode = (
+            <div className="flex relative" style={{ color: this.props.menuIconStyle.color }}>
```

Comment:
We've started using a `Container` component for this kind of stuff. Trying to move away from in-line styles. Not a huge change... Just using this `Container` component with some common sense layout / size props and then a `Text` component to wrap text with some common sense text props.

Reply:
Why are we trying to move away from in-line styles? Is it because it's easier to read props?

Reply:
Also, don't see `flex` as an option in the Container component. Should I add it? 

Looking at the `.flex` class from [BassCss](http://basscss.com/), it applies:

```
display: -ms-flexbox;
display: flex;
```

So it's doing lots of browser compatibility under-the-hood. Are we sure we want to forgo that?

Reply:
Looking at the `Container` component, I realize it's still using inline styles and should probably be changed to be using styled-components, which does automatically add browser prefixes. I can do this.

You can still add classNames to `Container` component which is what we've been doing to get `flex` layouts. Agree that the separation of concerns there is a bit confusing (ie. when to use props, vs when to use basscsss classnames). As mentioned, we've been trying to stay away from inline styles and using basscss where convenient. In an ideal world, I think Container would have the basscss features as typed props (https://jxnblk.com/rebass/). The reasoning behind this is: it's nicer and more "Reacty", our dependencies on styles are more explicit and specific (no dependency on global stylesheet) so if we ever want to move some components to a re-usable package for example, the user of that package wouldn't also have to include rebass. 

Anyway, we're probably not going to do this package thing, but I think trying to remove inline styles is a no brainer :P